### PR TITLE
LRDOCS-4928, What's New with Liferay Forms

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-4928

### CODE REVIEW

File: discover/portal/articles/130-forms/01-whats-new.markdown [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+: By default, a Form is autosaved every minute (without the Form builder
+clicking *Save*). The duration is configurable in Control Panel &rarr;
+Configuration &rarr; System Settings &rarr; Forms. Autosaves do not propagate
+to published Forms. 
+
+![Figure x: ](../../images/forms-autosave-interval.png)
+
+**Autocomplete Text Fields**
+: Using a Data Provider or manually typed options, configure a Text field to
+offer Users automcompleted values to choose from as they type in the Text field.
```
Comment (natocesarrego) [inline discover/portal/articles/130-forms/01-whats-new.markdown]: Typo in autocompleted.

File: discover/portal/articles/130-forms/01-whats-new.markdown [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+: Add validation logic to a Text field and create custom messages for form
+validation errors.
+<!-- I don't see RegEx Validation. Did that not make it into 7.1-->
+
+![Figure x: Validate Text fields to ensure Users fill in the correct data.](../../images/forms-text-validation.png)
+
+**Data Provider Enhancements**
+: Data Providers are still used to populate Select from List fields. Now, the
+Data Provider setup form has been overhauled, and you can now import and export
+Data Provider definitions. Data Providers are also linked with the Auotcomplete
```
Comment (natocesarrego) [inline discover/portal/articles/130-forms/01-whats-new.markdown]: Typo Autocomplete.

File: discover/portal/articles/130-forms/01-whats-new.markdown [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+![Figure x: A Success Page provides feedback to the User that the form
+Submission was successful.](../../images/forms-success-page.png)
+
+**Duplicate Forms and Form Fields**
+: Duplicate a Form to avoid building a similar one from scratch. Duplicate Form
+Fields if you need multiple similarly configured fields.
+
+![Figure x: Duplicate a Form to get a head start on your next, similar Form.](../../images/forms-duplicate.png)
+
+**Validate Text Field Entries**
```
Comment (natocesarrego) [inline discover/portal/articles/130-forms/01-whats-new.markdown]: Actually, this is a feature already available on 7.0. The real new feature in 7.1 is the possibility to use regular expressions when the field _If input_ has the value _Does not match_. 😉

**Review by natocesarrego — COMMENTED** [review]
Hey @rbohl,

This article is fully reviewed. Please see my notes.

Thanks.

File: discover/portal/articles/130-forms/01-whats-new.markdown (hunk 1 above)
Comment (rbohl) [reply discover/portal/articles/130-forms/01-whats-new.markdown]: fixed in master

File: discover/portal/articles/130-forms/01-whats-new.markdown (hunk 2 above)
Comment (rbohl) [reply discover/portal/articles/130-forms/01-whats-new.markdown]: fixed in master

**Pull request merged**